### DISCUSSION TITLE
Calibration plot

### DISCUSSION THREAD

**Discussion by raminsalmas** [sebp/scikit-survival#283] [General]
I'm wondering it there's any function in the scikit-survival to generate 'calibration plot'?

**Comment by sebp** [comment]
PR #101 started to implement this, but it has been dormant for a while now.

**Comment by vinyasHarish95** [comment]
Thanks for your reply @sebp and all your work on scikit-survival. Is there cross-compatibility between model objects in scikit-survival and lifelines? lifelines has a [function to create calibration plots](https://lifelines.readthedocs.io/en/latest/Survival%20Regression.html#model-selection-and-calibration-in-survival-regression) and it's based on a [different implementation](https://github.com/CamDavidsonPilon/lifelines/blob/master/lifelines/calibration.py) than that proposed in [PR#101](https://github.com/sebp/scikit-survival/pull/101). 

I ask since I wish to check calibration visually for more complex models (e.g. random survival forests) that are supported in scikit-survival but not in lifelines.

**Reply by sebp** [reply]
@vinyasHarish95 Thanks for pointing out the function in lifelines. I'll have a look to check in which regards it differs.,

**Reply by vinyasHarish95** [reply]
Thanks @sebp - I have some bandwidth over the next few weeks to take a stab at implementation (but might need some help (e.g. around designing appropriate unit tests) as I've never contributed to an open source project before. 
The lifelines implementation is only ~100 LoC so does not seem so bad. 
Do let me know if you're interested in working on this - I can imagine other users would benefit from this feature.

**Reply by sebp** [reply]
It certainly would be nice to have and contributions are always welcome. Feel free to open a PR and I'd be happy to give feedback.
